config/project.ini

```diff
@@ -0,0 +1,251 @@
+
+## Site Settings
+ckan.site_url = http://127.0.0.1:5000
+ckan.user_reset_landing_page = /user/reset
+ckan.hide_version = True
+
+## QGOV Settings
+ckan.base_public_folder = public
+ckan.base_templates_folder = templates
+extra_public_paths =
+extra_template_paths =
+feedback_form_recipients =
+feedback_redirection = /article/thanks
+
+## Search Settings
+solr_url = http://127.0.0.1:8983/solr
+
+## Redis Settings
+ckan.redis.url = redis://127.0.0.1:6379/0
+
+## Google Analytics
+googleanalytics.id = UA-1111111-1
+ckan.data_qld_googleanalytics.id = UA-111111111-1
+## Google Tag Manager
+ckan.google_tag_manager.gtm_container_id = GTM-12345
+
+## ckanext-odi-certificates
+ckan.odi_certificates.dataset_base_url = https://dataqld-dev.galv1.links.com.au
+
+# YTP Comments
+ckan.comments.bad_words_file = /usr/lib/ckan/default/src/ckanext-ytp-comments/ckanext/ytp/comments/bad_words.txt
+
+## Storage Settings
+ckan.storage_path = %(here)s/storage
+
+## S3 Storage Settings
+ckanext.s3filestore.aws_use_ami_role = true
+ckanext.s3filestore.aws_bucket_name = bucket
+ckanext.s3filestore.addressing_style = virtual
+#ckanext.s3filestore.download_proxy = <PROXY>
+ckanext.s3filestore.region_name= ap-southeast-2
+ckanext.s3filestore.signature_version = s3v4
+ckanext.s3filestore.acl = ssm/s3filestore_acl
+ckanext.s3filestore.filesystem_download_fallback = True
+
+ckanext.cloudstorage.driver = S3_AP_SOUTHEAST2
+ckanext.cloudstorage.container_name = bucket
+ckanext.cloudstorage.driver_options = use_role
+ckanext.cloudstorage.use_secure_urls = 1
+
+# download resources via Squid so we can block internal and private addresses
+# ckan.download_proxy = http://localhost:3128
+
+#the plan; instead of on disk, we will be using the uploader, once we work out to get files dynamically from it
+ckanext-archiver.archive_dir = %(here)s/storage/resource_cache
+#the plan; drop this for patten /dataset/{id}/resource/{resource_id}/archive/{filename}
+ckanext-archiver.cache_url_root = /resource_cache
+ckanext-archiver.max_content_length = 250000000
+ckanext-archiver.user_agent_string = "CKAN archiver https://localhost/"
+ckanext-archiver.verify_https = True
+
+# Use these options in local environment if you don't want to configure S3
+# ckanext.ssm_config.region_name = ap-southeast-2
+# ckanext.s3filestore.check_access_on_startup = no
+
+###############################################################################
+#                          Environment settings: end                          #
+###############################################################################
+
+ckan.tracking_enabled = true
+
+## Session settings
+beaker.session.secure = true
+beaker.session.timeout = 3600
+beaker.session.save_accessed_time = true
+beaker.session.cookie_expires = true
+
+## datastore
+# We need the SQL endpoint, despite the non-ideal security implications
+ckan.datastore.sqlsearch.enabled = True
+ckan.datastore.sqlsearch.allowed_functions_file = %(here)s/allowed_functions.txt
```

Comment:
This would be a good place to document why we need to override the function list.

Reply:
This file was referred from the config when we got our copy or your environment. We've just added it to the repo, so that it won't get lost accidentally. Try asking your co-workers, why they added it initially

Reply:
@mutantsan please drop row 102 and [config/allowed_functions.txt](https://github.com/qld-gov-au/ckanext-data-qld/pull/175/files/4d2bbef004d16c6695c4761c99e0d580d5fb2fad#diff-23ab2b4cc1eea9edfdcda14a13c5b8a5fc0cf656b088e8fbf08f82937e968e53)